decred/decred/dcr/dcrdata.py

```diff
@@ -962,7 +953,7 @@ def confirmUTXO(self, utxo, block=None, tx=None):
             pass
         return False
 
-    def sendOutputs(self, outputs, keysource, utxosource, feeRate=None):
+    def sendOutputs(self, outputs, keysource, utxosource, relayFee):
```

Comment:
Looks like the argument documentation was already missing, but please add it while you're here.